README.md

```diff
@@ -1,2 +1,45 @@
-# Hello-World
-my new repository
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Hello-World
+
```

Comment:
- [ ] I think it would be better if you replace the **# Project Name** text with the project's title (e.g.: **# Hello Microverse**) to make your documentation looks professional.

- [ ] Also make sure to add a good description for your project to let users know what is about the project and as required for the project to have a description. You can add your description where you started: " > Hello-world...."(line 5)